### ISSUE TITLE
dotnet test on a multi-target projects logs only the last target

### ISSUE THREAD

**Issue by sidshetye** [spekt/LoggerExtensions#27]
This one is possibly related to [this vstest issue](https://github.com/Microsoft/vstest/issues/1603). Posting it here in case this is an issue with individual logger extensions rather than a more fundamental vstest issue (to be determined). If determined that the actual fix only lies in vstest, we can close this out.

-------
## Description
`dotnet test --logger:"nunit;LogFilePath=results.xml"` on a multi-target project (e.g. `<TargetFrameworks>netcoreapp2.0;net45</TargetFrameworks>` in the `.csproj`) causes the logger to only log the last target. Possibly because the previous targets' logger output file is overwritten by subsequent targets. 

## Steps to reproduce

`dotnet test --logger:"nunit;LogFilePath=results.xml"`

## Expected behavior
Either generate as many .xml files as there are targets (e.g. `results.net45.xml`, `results.netcoreapp2.0.xml`) OR assemble the results into the single .xml if file format allows for it. Perhaps the user can pick either of those behaviors.

## Actual behavior
Only a single `results.xml` is created and it's contents only hold the results of the last target framework in the `.csproj`.

## Environment
Has the following inside the test it's `.csproj`
```
<PackageReference Include="NUnit" Version="3.9.0" />
<PackageReference Include="NUnit3TestAdapter" Version="3.9.0" />
<PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.3.0" />
<PackageReference Include="NunitXml.TestLogger" Version="1.1.0" />
``` 
OS: Windows 10 v1803
.NET SDK: `dotnet --version` shows `2.1.0`
Not sure if/how dotnet test wraps over vstest but this is `vstest.console.exe`
```
C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\Common7\IDE\CommonExtensions\Microsoft\TestWindow>vstest.console.exe
Microsoft (R) Test Execution Command Line Tool Version 15.7.1
Copyright (c) Microsoft Corporation.  All rights reserved.
```

**Comment by codito** [comment]
@sidshetye thanks for reporting this issue. It seems similar to #24. I think @mcblair is working on a fix in the PR #25.

**Comment by codito** [comment]
Closing this as duplicate of #24.
